packages/dev/core/src/Engines/thinEngine.functions.ts

```diff
@@ -265,6 +265,7 @@ export function _preparePipelineContext(
     defines: Nullable<string>,
     transformFeedbackVaryings: Nullable<string[]>,
     _key: string = "",
+    onReady: () => void,

```

Comment:
@sebavan: making sure this is ok to be here before the injections?

Reply:
Lets check with @RaananW who extracted this for precompilation ?

Reply:
As long as it is consistent across engines I don't see why not. Also need to be sure that when this function is injected the function is passed correctly.